src/python/dxpy/dx_extract_utils/InputsValidator.py

```diff
@@ -81,22 +85,23 @@ def interpret_conditions(self):
                 method_to_call = self.conditions_funcs.get(condition)
                 getattr(self, method_to_call)(key)
 
-    def throw_message(self, check):
+    def throw_schema_message(self, check):
         type = self.schema.get(check).get("error_message").get("type")
+        message = self.schema.get(check).get("error_message").get("message")
         if type == "warning":
-            self.throw_warning(check)
+            self.throw_warning(message)
         elif (type == None) or (type == "error"):
```

Comment:
A bit safer perhaps using `isinstance`:

```suggestion
        elif (isinstance(type, type(None)) or (type == "error"):
```